Cakes-Co/index.html

```diff
@@ -1,16 +1,118 @@
 <!DOCTYPE html>
 <html lang="en">
-
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <title>Responsive Cake webpage</title>
     <!-- Add a link to your css file here -->
-</head>
+    <link rel="stylesheet" href="style.css" />
+  </head>
 
-<body>
-    <!-- Add your markup here -->
-</body>
+  <body>
+    <div class="layout">
+      <!-- HEADER SECTION -->
+      <header>
+        <div class="logo-hamburger-container">
+          <img
+            src="/Cakes-Co/images/cupcake-svg-download.jpg"
+            width="100px"
+            height="100px"
+            class="cupcake-logo"
+            alt="cake-picture"
```

Comment:
this is fine, but remember the alt attribute can take free form text, so no need to hypenate it. A slightly improved alt attribute might be `alt="a picture of a cake"`